server/server.go

```diff
@@ -238,6 +238,19 @@ func (u *UpdateServer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if params.OS == "darwin" {
+		v360, err := semver.Make("3.6.0")
+		if err != nil {
+			log.Debugf("semver.Make: %v", err)
+			u.closeWithStatus(w, http.StatusNoContent)
+		}
+		currentVersion, err := semver.Parse(params.AppVersion)
```

Comment:
`err` is not checked. If error happens, `currentVersion` will be `Version{}` which should be always `LT(v360)`, but we'd better to make it explicit.